### ISSUE TITLE
readthedocs versioning 

### ISSUE THREAD

**Issue by tikal004** [DCAN-Labs/data-processing-handbook#111]
We have 3 versions of this readthedocs active,

- latest (default)
- stable
- doc-dev

The doc-dev branch on github was recently deleted.
stable is tied to the most recent tag, which is quite outdated.

**This is a simple fix, but warrants a discussion about workflow.** 
We can probably get rid of stable, this isn't the kind of codebase that needs tagged releases.
How do we want to use doc-dev? Should we have a more official approval process for reviewing pull request?

**Comment by hough129** [comment]
@tikal004 from what I can remember, I don't think we can delete stable? Stable also only get pushed when we tag new versions, so maybe we should discuss when a new version is warranted.

I would delete doc-dev. I deleted the doc-dev branch when I merged it in, because I realized that the entire repo is doc-dev, and I want people to be able to work simultaneously. 


In terms of pull request approval, I just make sure whatever someone wrote makes sense, but I am open to ideas.
